chrome/content/zotero/xpcom/data/item.js

```diff
@@ -868,10 +868,8 @@ Zotero.Item.prototype.setField = function(field, value, loadIn) {
  * Get the title for an item for display in the interface
  *
  * This is the same as the standard title field (with includeBaseMapped on)
- * except for letters and interviews, which get placeholder titles in
- * square braces (e.g. "[Letter to Thoreau]"), and cases
```

Comment:
Sorry for the late reply.

In my test results, this comment is no longer accurate, as in the letter item below, `getDisplayTitle` returns the title, not the placeholder `[Letter to author]`.

![image](https://github.com/zotero/zotero/assets/44738481/5cd3262c-8243-43c1-b720-67b94bb2e508)

This is also obvious from the source code of `getDisplayTitle`:

```js
Zotero.Item.prototype.getDisplayTitle = function () {
	if (this._displayTitle !== null) {
		return this._displayTitle;
	}
	return this._displayTitle = this.getField('title', false, true);
}
```

The source code no longer contains the code of the `[]` placeholder, and there isn't even code to determine the item type,  which was the case in previous versions:

![image](https://github.com/zotero/zotero/assets/44738481/26564d60-66a8-4f75-bada-1d20dab51154)

Am I misunderstanding somewhere?

Reply:
Oh sorry, I wasn't reading carefully and I didn't notice that `updateDisplayTitle()` had been called in `items.js`, making `item._displayTitle!==null` (Before that I thought updateDisplayTitle was a useless method of historical legacy).

I'd like to bother again:

For letters, etc., when the user modifies the author field, `updateDisplayTitle` should be triggered immediately and refresh the itemTree, as in the following recording, when the Creators are changed, the displayTitle in the itemTree is not updated (even after switching the Collection). 

I think this could lead to confusion. And at this point `item.getDisplayTitle()` is the old value, which may cause the plugin to handle it incorrectly(?).

When `item.updateDisplayTitle()` is called manually, and Collection is switched again, the displayTitle in itemTree is only updated.

https://github.com/zotero/zotero/assets/44738481/28b028c5-10de-4acb-adac-5d3e508c5ed3